### PR TITLE
RBTA-140/bug-form-buttons/frankiv

### PR DESCRIPTION
**Pull request by oksanafrankiv**
https://jira.pasv.us/browse/RBTA-140

### CODE REVIEW

File: test/specs/reg/bug-form/bug-form-buttons.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import {assert} from 'chai';
+import sel from '../../../selectors/bug-form';
+import exp from '../../../expected/bug-form';
+import help from "../../../helpers/helpers";
+
+
+describe('General', function () {
+    it('Font Family', function () {
+        browser.url('/');
+        help.login();
```
Comment (vianat) [inline test/specs/reg/bug-form/bug-form-buttons.js]: you can remove string 9

Comment (artsenius) [reply test/specs/reg/bug-form/bug-form-buttons.js]: agree

File: test/specs/reg/bug-form/bug-form-buttons.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+describe('Submit', function () {
+    it('Background Color', function () {
+        let subBackColor = $(sel.submitBut).getCSSProperty('background-color').parsed.hex;
+        assert.equal(subBackColor, exp.subBackColor);
+    });
+
+    it(' Hover Background Color', function () {
+        $(sel.submitBut).moveTo();
+        browser.pause(500);
```
Comment (artsenius) [inline test/specs/reg/bug-form/bug-form-buttons.js]: try to put less ms, for example 200-300. If it enough to change to hover state, then we save a 50% of time.

File: test/specs/reg/bug-form/bug-form-buttons.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+describe('Clear Form Button', function () {
+    it('Background Color', function () {
+        let clearBackColor = $(sel.clearBut).getCSSProperty('background-color').parsed.hex;
+        assert.equal(clearBackColor, exp.clearBackColor);
+    });
+
+    it(' Hover Background Color', function () {
+        $(sel.clearBut).moveTo();
+        browser.pause(500);
```
Comment (artsenius) [inline test/specs/reg/bug-form/bug-form-buttons.js]: try to put less ms, for example 200-300. If it enough to change to hover state, then we save a 50% of time.

**Pull request merged**